blocks/library/image/index.js

```diff
@@ -19,21 +19,26 @@ registerBlock( 'core/image', {
 		caption: html( 'figcaption' )
 	},
 
-	edit( { attributes, isSelected, setAttributes } ) {
+	edit( { attributes, setAttributes, focus, updateFocus } ) {
 		const { url, alt, caption } = attributes;
+		const focusCaption = () => updateFocus( { editable: 'caption' } );
 
+		/* eslint-disable */
```

Comment:
Also, we should be specifically disable only the lint rule we want to target, in case there are other lint issues in the region.

I like the idea of commenting on any disable we apply, since it holds us accountable to understanding why lint errors occur and how they're not applicable.